ern-api-gen/src/models/properties/ObjectPropertyBase.ts

```diff
@@ -39,7 +39,9 @@ export class ObjectPropertyBase extends Property {
     if (!properties) {
       this._additionalProperties = null
     }
-    if (properties.type) {
+    // @todo the ref support here is only dangerously supported
+    // { $ref: '#/definition/ModelName }
+    if (properties.type || properties.$ref || properties.genericRef) {
```

Comment:
_this works_ and clearly has _not been a problem for other users of this api_.  this is unlikely to break other consumers, just _enable_ previously-partially-supported-functionality to them/me